core/src/main/scala/io/udash/generator/plugins/scalacss/ScalaCSSDemosPlugin.scala

```diff
@@ -90,15 +215,15 @@ object ScalaCSSDemosPlugin extends GeneratorPlugin with SBTProjectFiles with Fro
          |
          |  private val content = div(
          |    LocalStyles.render[TypedTag[org.scalajs.dom.raw.HTMLStyleElement]],
-         |    div(
+         |    h2(
          |      "You can find this demo source code in: ",
          |      i("${settings.rootPackage.mkPackage()}.${settings.viewsSubPackage.mkPackage()}.DemoStylesView")
          |    ),
          |    h3("Example"),
          |    p(LocalStyles.redItalic)("Red italic text."),
          |    p(LocalStyles.obliqueOnHover)("Hover me!"),
          |    h3("Read more"),
-         |    a(href := "http://japgolly.github.io/scalacss/book/")("Read more in ScalaCSS docs.")
+         |    a(DemoStyles.underlineLinkBlack)(href := "http://japgolly.github.io/scalacss/book/")("Read more in ScalaCSS docs.")
```

Comment:
We should add here also Udash Guide link: http://guide.udash.io/#/frontend/templates